include/sway/config.h

```diff
@@ -260,7 +260,7 @@ struct bar_config {
 
 #if HAVE_TRAY
 	char *icon_theme;
-	const char *tray_bindings[10]; // mouse buttons 0-9
+	list_t *tray_bindings; // struct tray_binding *
```

Comment:
How about wl_list instead?

Reply:
Uh sure, I can change it to use wl_list. I just used list_t since that's what all the other bindings are using

Reply:
Yeah, I should have caught this in the other patches. We should be moving away from list_t.